### PR TITLE
Miscellaneous Section on Guides

### PR DESCRIPTION
**Pull request by Circleen**
Was originally closed because of upriver.

### CODE REVIEW

File: wiki/Guides/en.md [hunk 1]
```diff
@@ -49,3 +48,10 @@ Below is a sorted list of guides created by members of the osu!community; most o
 - [Cropping with Complex Backgrounds](Cropping_with_Complex_Backgrounds)
 - [Cropping with Simple Backgrounds](Cropping_with_Simple_Backgrounds)
 - [Making Properly Centered Spinners](Making_Properly_Centered_Spinners)
+
+## Miscellaneous
+
+- [Recording osu!](Recording_osu!)
```
[the remark this answers is not in the document]
Comment (Circleen) [reply wiki/Guides/en.md]: Will do

File: wiki/Guides/en.md [hunk 2]
```diff
@@ -35,7 +35,6 @@ Below is a sorted list of guides created by members of the osu!community; most o
 - [GL Support Issues](GL_Support_Issues)
```
[the remark this answers is not in the document]
Comment (Circleen) [reply wiki/Guides/en.md]: Ok.

**Comment by Circleen** [comment]
Finished?

File: wiki/Guides/en.md [hunk 3]
```diff
@@ -49,3 +47,10 @@ Below is a sorted list of guides created by members of the osu!community; most o
 - [Cropping with Complex Backgrounds](Cropping_with_Complex_Backgrounds)
 - [Cropping with Simple Backgrounds](Cropping_with_Simple_Backgrounds)
 - [Making Properly Centered Spinners](Making_Properly_Centered_Spinners)
+
+## Miscellaneous
+
+- [Discord RPC](Discord_RPC)
+- [GL Support Issues](GL_Support_Issues)
+- [Live-streaming osu!](Live_streaming_osu!)
```
Comment (MegaApplePi) [inline wiki/Guides/en.md]:
```suggestion
- [Live Streaming osu!](Live_Streaming_osu!)
```

**Comment by Circleen** [comment]
Ready to merge yet?

File: wiki/Guides/en.md [hunk 4]
```diff
@@ -49,3 +47,10 @@ Below is a sorted list of guides created by members of the osu!community; most o
 - [Cropping with Complex Backgrounds](Cropping_with_Complex_Backgrounds)
 - [Cropping with Simple Backgrounds](Cropping_with_Simple_Backgrounds)
 - [Making Properly Centered Spinners](Making_Properly_Centered_Spinners)
+
+## Miscellaneous
+
+- [Discord RPC](Discord_RPC)
```
Comment (peppy) [inline wiki/Guides/en.md]: this is wrong. it should be "Rich Presence".

it's not RPC. that's a developer term which shouldn't

File: wiki/Guides/en.md [hunk 5]
```diff
@@ -49,3 +47,10 @@ Below is a sorted list of guides created by members of the osu!community; most o
...
+- [GL Support Issues](GL_Support_Issues)
```
Comment (peppy) [inline wiki/Guides/en.md]: what does this even mean? how about OpenGL Issues? GL on its own means nothing

**Pull request merged**